src/api/zoops.ts

```diff
@@ -0,0 +1,33 @@
+import express from "express";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+const zoopsRouter = express.Router();
+
+// GET /api/zoops
+
+zoopsRouter.get("/", async (req, res, next): Promise<void> => {
+    try {
+        const zoops = await prisma.zoop.findMany();
+        res.send({zoops});
+    } catch (e) {
+        next(e);
+    }
+})
+
+// POST /api/zoops
+
+zoopsRouter.post("/", async (req, res)=> {
```

Comment:
Wait, this route will require Auth protection - please add a TODO to require logged in user for this request. Also, it will NOT receive authorId in the body

Reply:
So `authorId` will be in the request headers, right? Will it be in `req.headers.authorization` like the web token?

Reply:
No, it will not be in the request header. You will add it in your authentication middleware by adding the user.

So whoever the logged in user (stored in `req.user`) will be the author ID

Reply:
Got it. So there is a user property on req or it gets created when the server returns a user object to client?